hardware_interface/include/hardware_interface/robot_hw.h

```diff
@@ -123,7 +124,9 @@ class RobotHW : public InterfaceManager
 
     return in_conflict;
   }
-/** \name Hardware Interface Switching
+  /**\}*/
+
+  /** \name Hardware Interface Switching
```

Comment:
The [doxygen group](http://www.doxygen.nl/manual/grouping.html#memgroup) "Hardware Interface Switching" looks different than the others ("Control Loop" and "Resource Management"). This happens probably because of the `enum SwitchState` mentioned in [this discussion](https://github.com/ros-controls/ros_control/pull/433#discussion_r405916399).